src/WebRTC/Browser.WebRTC.fs

```diff
@@ -339,7 +339,7 @@ type RTCRtpStreamStats =
 type RTCReceivedRtpStreamStats =
     inherit RTCRtpStreamStats
     abstract packetsReceived: uint32
-    abstract packetsLost: int64
+    abstract packetsLost: uint32
```

Comment:
This has made me realize there are other properties in the WebRTC bindings typed as `int64`. This may not work well because Fable doesn't translate `int64` as native JS number. Maybe the other properties typed as `int64` should change to `uint32` as well?